markets/bfp-market/contracts/modules/MarketConfigurationModule.sol

```diff
@@ -49,10 +50,14 @@ contract MarketConfigurationModule is IMarketConfigurationModule {
         OwnableStorage.onlyOwner();
 
         PerpMarketConfiguration.Data storage config = PerpMarketConfiguration.load(marketId);
-
+        PerpMarketConfiguration.GlobalData storage globalConfig = PerpMarketConfiguration.load();
         // Only allow an existing per market to be configurable. Ensure it's first created then configure.
         PerpMarket.exists(marketId);
 
+        if (data.minMarginUsd < globalConfig.maxKeeperFeeUsd) {
+            revert ErrorUtil.InvalidMinMargin();
```

Comment:
Can you use the same revert `error InvalidParameter(string parameter, string reason);`?